linera-service/src/cli_wrappers/local_net.rs

```diff
@@ -589,21 +493,14 @@ impl LocalNet {
     }
 
     async fn initialize_storage(&mut self, validator: usize, shard: usize) -> Result<String> {
-        let shard_str = match self.database {
-            Database::RocksDb => format!("_{}", shard),
-            _ => String::new(),
-        };
-        let namespace = format!("{}_server_{}{}_db", self.table_name, validator, shard_str);
+        let namespace = format!("{}_server_{}_{}_db", self.table_name, validator, shard);
```

Comment:
The shards must all use the same namespace. That's why in the non-RocksDB case the `shard_str` was always empty.